account_central_billing/README.rst

```diff
@@ -22,7 +22,7 @@ There are no special installation instructions for this module.
 Configuration
 =============
 
-#. For each centrally billed partner, select the approriate supplier or customer billing option.
+#. For each centrally billed partner, select the appropriate supplier or customer billing option.
```

Comment:
Also in this README is a link to report issues, please update to correct repository